### PR TITLE
feat: Add WebOTX to JEE8 Compatibible products

### LINKED ISSUE #1626: Compatible Software Product Request for WebOTX Application Server V11 for Full Platform 8.0
**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
https://www.eclipse.org/membership/showMember.php?member_id=1399

**A product name and version**
WebOTX Application Server V11

**The Profile (Full Platform, Web Profile, Core Profile) that the Compatible Product is submitted to be listed under**
Full Platform 8.0

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
WebOTX_logo

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
NEC Corporation

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
https://jpn.nec.com/webotx/appserver/ee8_tck_result_11100000.html

**The URL where Compatible Software Product is available for download**
https://jpn.nec.com/webotx/appserver/index.html

### PR DESCRIPTION
**Pull request by ZacharySabourin**
Resolves https://github.com/jakartaee/jakarta.ee/issues/1626

### CODE REVIEW

**Review by ericpoirier — APPROVED** [review]

**Review by oliviergoulet5 — APPROVED** [review]

**Comment by chrisguindon** [comment]
@ZacharySabourin Before you can merge this, we will want a +1 from Tanya to make just it's something we want to add on our site.

Here's a link to what @oliviergoulet5 did last time:
https://github.com/jakartaee/jakarta.ee/issues/1596#issuecomment-1380614462

**Pull request merged**
